src/main.py

```diff
@@ -141,20 +148,23 @@ def main():
                 last_name=input("last name: ")
                 username2 = look_value(username)
                 password = getpass()
+                logedin = 0
                 check_pw(password)
-                data_entry(username2, password,first_name,last_name)
+                data_entry(username2,password,first_name,last_name,logedin)
             elif capacity == 5:
                 print("The amount of allowed accounts (5) has been reached")
                 continue
 
-
     #LOGIN TO PROGRAM
         elif choice == 'l':
             username = input("Username: ")
             password = getpass()
             isLoggedIn = login_attempt(username, password)
+            
 
             if isLoggedIn:
+                
+                login(username)
```

Comment:
If we're storing this on the db then we'll also need to reset the value (log out a user) on program end or sign out.

Reply:
Yeah we do! but I forgot to added to the logout function that was my bad